angular-oauth2-oidc/src/oauth-service.ts

```diff
@@ -312,6 +311,10 @@ export class OAuthService
         this._storage = storage;
         this.configChanged();
     }
+	
+	public getStorage(): OAuthStorage {
+		return this._storage;
+	}
 
```

Comment:
Would this be better suited as a property?

```ts
public get storage() : OAuthStorage {
   return this._storage;
}
```

Reply:
maybe, but setStorage is used as method
that's why i also use it like this